build/build_config.h

```diff
@@ -254,13 +254,13 @@
 #define BUILDFLAG_INTERNAL_IS_OZONE() (0)
 #endif
 
-#if defined(IS_COBALT)
+#if defined(ENABLE_BUILDFLAG_IS_COBALT)
```

Comment:
I submitted this change, could you rebase your PR so this diff disappears?

Reply:
Done